### PR TITLE
cleanup(storybook): shorten globs for tests and stories

### PR DESCRIPTION
**Pull request by mandarini**
Shorten globs for test files and stories files.

### CODE REVIEW

File: packages/storybook/src/generators/configuration/configuration.spec.ts [hunk 1]
```diff
@@ -61,17 +61,8 @@ describe('@nrwl/storybook:configuration', () => {
       );
 
       expect(
-        storybookTsconfigJson.exclude.includes('../**/*.spec.ts')
+        storybookTsconfigJson.exclude.includes('../**/*.spec.{ts,tsx,js,jsx}')
```
Comment (AgentEnder) [inline packages/storybook/src/generators/configuration/configuration.spec.ts:64]: This one looks like it should only be .ts, the previous tests were expecting .tsx, .js, and .jsx to not be included.

Comment (mandarini) [reply packages/storybook/src/generators/configuration/configuration.spec.ts:64]: Yeah, I know, but I think there's no reason to just be `.ts`, now that I revisited that old code... 😬

File: packages/storybook/src/generators/configuration/util-functions.ts [hunk 2]
```diff
@@ -166,15 +166,12 @@ export function configureTsProjectConfig(
...
+    !tsConfigContent?.exclude?.includes('**/*.stories.{ts,tsx,js,jsx,mdx}')
   ) {
     tsConfigContent.exclude = [
       ...(tsConfigContent.exclude || []),
-      '**/*.stories.ts',
-      '**/*.stories.js',
-      ...(isFramework('react', schema) || isFramework('react-native', schema)
-        ? ['**/*.stories.jsx', '**/*.stories.tsx']
-        : []),
+      '**/*.stories.{ts,tsx,js,jsx,mdx}',
```
Comment (AgentEnder) [inline packages/storybook/src/generators/configuration/util-functions.ts:174]: Maybe something like 
```typescript
const extensions = isFramework('react', schema) || isFramework('react-native', schema) ? ['ts','tsx','jsx','js'] : ['ts', 'js'];
tsConfigContent.exclude = [
  (...tsConfigContent.exclude || []),      
  `**/*.stories.{${extensions.join(',')}}`,
]
```

**Review by AgentEnder — CHANGES_REQUESTED** [review]

File: packages/storybook/src/generators/configuration/configuration.spec.ts (hunk 1 above)
Comment (AgentEnder) [reply packages/storybook/src/generators/configuration/configuration.spec.ts:64]: I think it makes sense to narrow it down to ts/js only if not react at least, maybe something like below suggestion?

File: packages/storybook/src/generators/configuration/util-functions.ts (hunk 2 above)
Comment (mandarini) [reply packages/storybook/src/generators/configuration/util-functions.ts:174]: Yeah, I decided that there's no real value in so much logic behind these. We want to exclude tests, include stories in `.storybook/tsconfig.json` and we want to exclude stories in the project's `tsconfig.json`. No need for framework specific things. That's why I made it all like that.

File: packages/storybook/src/generators/configuration/configuration.spec.ts (hunk 1 above)
Comment (mandarini) [reply packages/storybook/src/generators/configuration/configuration.spec.ts:64]: Yeah ok makes sense!

**Comment by mandarini** [comment]
Updated the logic! :D

**Comment by mandarini** [comment]
@AgentEnder do you think that looks good for now?

**Comment by AgentEnder** [comment]
Yeah the changeset looks fine to me, we should do some manual testing to ensure the shortened globs are being used by tsc correctly though. 

Also failing e2e.

**Comment by mandarini** [comment]
Ok, will do manual testing!!!

**Comment by mandarini** [comment]
So, after all, the `.{ts,tsx}` syntax does not work for `tsonfig` files. :( Closing PR.

**Pull request closed without merging**